packages/@aws-cdk/aws-apigatewayv2/lib/http/authorizer.ts

```diff
@@ -58,6 +81,39 @@ export interface HttpAuthorizerProps {
    * @default - required for JWT authorizer types.
    */
   readonly jwtIssuer?: string;
+
+  /**
+   * Specifies whether a Lambda authorizer returns a response in a simple format.
+   *
+   * By default, a Lambda authorizer must return an IAM policy.
+   * If enabled, the Lambda authorizer can return a boolean value instead of an IAM policy.
+   *
+   * @default - required for Request authorizer types
```

Comment:
Let's pick a default for Request authorizer types as well. What is the default that CF applies for request authorizer types?

Reply:
CF doesn't specify this by default, it's an optional property.

Reply:
So, will the deployment will fail if this is not specified for request authorizers? 
If it's optional, either CF picks a default or there is an error during deployment.

If CF doesn't pick a default, we should and I would suggest we go with `false`.

Reply:
Okay did some tests. The only time deployment fails is when authorizerPayloadVersion is not supplied. This is the only requirement for lambda authorizers.

So setting a default for `enableSimpleResponses` isn't required. cc @kbanman